internal/pkg/flink/internal/store/.snapshots/TestProcessSetStatement-should_return_all_keys_and_values_from_config_if_configKey_is_empty_after_updates

```diff
@@ -0,0 +1,99 @@
+(*types.StatementResults)({
+  Headers: ([]string) (len=2) {
+    (string) (len=3) "Key",
+    (string) (len=5) "Value"
+  },
+  Rows: ([]types.StatementResultRow) (len=7) {
+    (types.StatementResultRow) {
+      Operation: (types.StatementResultOperation) +I,
+      Fields: ([]types.StatementResultField) (len=2) {
+        (types.AtomicStatementResultField) {
+          Type: (types.StatementResultFieldType) (len=7) "VARCHAR",
+          Value: (string) (len=7) "catalog"
+        },
+        (types.AtomicStatementResultField) {
+          Type: (types.StatementResultFieldType) (len=7) "VARCHAR",
+          Value: (string) ""
+        }
+      }
+    },
+    (types.StatementResultRow) {
+      Operation: (types.StatementResultOperation) +I,
+      Fields: ([]types.StatementResultField) (len=2) {
+        (types.AtomicStatementResultField) {
+          Type: (types.StatementResultFieldType) (len=7) "VARCHAR",
+          Value: (string) (len=16) "default_database"
+        },
+        (types.AtomicStatementResultField) {
+          Type: (types.StatementResultFieldType) (len=7) "VARCHAR",
+          Value: (string) ""
+        }
+      }
+    },
+    (types.StatementResultRow) {
+      Operation: (types.StatementResultOperation) +I,
+      Fields: ([]types.StatementResultField) (len=2) {
+        (types.AtomicStatementResultField) {
+          Type: (types.StatementResultFieldType) (len=7) "VARCHAR",
+          Value: (string) (len=8) "location"
+        },
+        (types.AtomicStatementResultField) {
+          Type: (types.StatementResultFieldType) (len=7) "VARCHAR",
+          Value: (string) (len=3) "USA"
+        }
+      }
+    },
+    (types.StatementResultRow) {
+      Operation: (types.StatementResultOperation) +I,
+      Fields: ([]types.StatementResultField) (len=2) {
+        (types.AtomicStatementResultField) {
+          Type: (types.StatementResultFieldType) (len=7) "VARCHAR",
+          Value: (string) (len=15) "org-resource-id"
+        },
+        (types.AtomicStatementResultField) {
+          Type: (types.StatementResultFieldType) (len=7) "VARCHAR",
+          Value: (string) ""
+        }
+      }
+    },
+    (types.StatementResultRow) {
+      Operation: (types.StatementResultOperation) +I,
+      Fields: ([]types.StatementResultField) (len=2) {
+        (types.AtomicStatementResultField) {
+          Type: (types.StatementResultFieldType) (len=7) "VARCHAR",
+          Value: (string) (len=13) "pipeline.name"
```

Comment:
Maybe update these old tests to use something like `table.results-timeout` or other key instead of `pipeline.name` just to avoid confusion. The latter doesn't exist anymore, and the user can't pick the pipeline name